src/tests/integration/test_mssql.py

```diff
@@ -112,3 +113,17 @@ def test_sqlplot_boxplot(ip_with_MSSQL, cell):
     out = ip_with_MSSQL.run_cell(cell)
 
     assert type(out.result).__name__ in {"Axes", "AxesSubplot"}
+
+
+def test_unsupported_function(ip_with_MSSQL, test_table_name_dict):
+    # clean current Axes
+    plt.cla()
+    out = ip_with_MSSQL.run_cell(
+        f"%sqlplot boxplot --table " f"{test_table_name_dict['taxi']} --column x"
+    )
+    assert isinstance(out.error_in_exec, UsageError)
+    assert "Ensure that percentile_disc function is available" in str(out.error_in_exec)
+    assert (
+        "If you need help solving this issue, "
+        "send us a message: https://ploomber.io/community" in str(out.error_in_exec)
+    )
```

Comment:
Do you think we should add a successful test where we use percentile_disc without getting this error?

Reply:
This is the success testcase:

https://github.com/ploomber/jupysql/blob/947af610def13a40bbbd4257a1a4a67cb540c9ed/src/tests/integration/test_generic_db_operations.py#L306

Reply:
Yes, I thought maybe we can create a scoped test to make sure that `_summary_stats` runs fluently with `percentile_disc` since `test_sqlplot_boxplot` may fail for other reasons. I guess we can skip it.

@idomic what do you think?
if not, we can merge it from my side.